sample/src/main/java/org/aerogear/graphqlandroid/data/UserData.kt

```diff
@@ -17,6 +19,11 @@ class UserData(val context: Context) {
 
     val noteslist = arrayListOf<Task>()
 
+    companion object {
+
+        val offlineArrayList = arrayListOf<com.apollographql.apollo.api.Mutation<Operation.Data,Void, Operation.Variables>>()
+    }
+
     var apolloQueryWatcher: ApolloQueryWatcher<AllTasksQuery.Data>? = null
```

Comment:
How this will work in generic use case. We use here all TasksQuery.Data

Reply:
@wtrocki In generic, the user will have to provide the query he/she wants to keep a watch for (Re-fetches watched GraphQL query). 

So in our library, either we can skip applying a watcher on the mutation, and this can be done on the client side depending on the client's use case (we will document them in our library).

Reply:
What do you think?